changelog/fragments/deprecate-crdversionflag.yaml

```diff
@@ -0,0 +1,8 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      For Ansible/Helm based-operators (ansible/v1, helm/v1), deprecate the flag `crd-version` since
+      `v1beta1` APIs is not longer offered from k8s 1.22 and this flag is deprecated for Golang.
```

Comment:
Do we need to back port this to the SDK release which supports k8s 1.22?

Reply:
It is not a bug fix so if we not be able to backport that is 100% fine.
However, I think that would be cool if we are able to do that